### PR TITLE
#162948864 Create an empty react app

### PR DESCRIPTION
**Pull request by Ruiru11**
#### What does this PR do?
- Setup an empty react project
- Setup Airbnb as our default lint guide
#### Description of Task to be completed?
Setup the basic structure of the frontend 
-Create new app by running`npx create-react-app <name_of_app>` then `cd <name_of_app>`
#### How should this be manually tested?
- Clone the repository `git clone`
- git checkout `ch-create-react-app-162948864`

- To check for linting errors run `npm run lint`

-  To run the app run `npm start` 

#### What are the relevant pivotal tracker stories?
[#162948864](https://www.pivotaltracker.com/story/show/162948864)

### CODE REVIEW

**Comment by tesh254** [comment]
The Readme file does not state how we can lint the application

**Review by tesh254 — COMMENTED** [review]
Update readme for all commands that are stated in the package.json like the `npm run lint` one may not know if it runs the lint

**Review by Deekerubo — APPROVED** [review]

**Review by tesh254 — APPROVED** [review]

**Comment by boltonotieno** [comment]
@tesh254  Changes have been made to the PR as requested

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (Betty-Kebenei) [inline public/index.html]: Change this to `Authors Haven `

**Review by danuluma — APPROVED** [review]
LGTM

**Comment by jamesbeamie** [comment]
Looks good

**Review by jamesbeamie — APPROVED** [review]

File: public/index.html (hunk 1 above)
Comment (boltonotieno) [reply public/index.html]: This has been worked on.

**Review by boltonotieno — APPROVED** [review]
LGTM

**Pull request merged**